### PR TITLE
Several PHP 7.4 fixes

### PR DESCRIPTION
**Pull request by brainfoolong**
On a fresh 3.13.0 install with php 7.4.0 following error occurs when tracker does track a page view, when using Mysqli as database driver. 

`Notice: Trying to access array offset on value of type int in /srv/0x.at/matomo/core/Tracker/Db/Mysqli.php on line 50`

This fix this error

### CODE REVIEW

File: core/DataTable/Filter/ColumnDelete.php [hunk 1]
```diff
@@ -161,7 +161,7 @@ protected function removeColumnsFromTable(&$table)
             }
             foreach ($this->columnsToRemove as $column) {
 
-                if (!array_key_exists($column, $row)) {
+                if (!property_exists($column, $row)) {
```
Comment (tsteur) [inline core/DataTable/Filter/ColumnDelete.php]: Could you maybe undo this change? We will investigate this as part of https://github.com/matomo-org/matomo/issues/15210

Comment (brainfoolong) [reply core/DataTable/Filter/ColumnDelete.php]: Yes, done.

**Comment by tsteur** [comment]
Awesome, thanks for quick change. Very appreciated.

**Review by tsteur — APPROVED** [review]
Need to wait for tests before merging though
